src/main/java/xyz/nucleoid/plasmid/game/player/GamePlayerJoiner.java

```diff
@@ -40,15 +40,9 @@ private static Set<ServerPlayerEntity> collectPlayersForJoin(ServerPlayerEntity
     private static Results tryJoinAll(Collection<ServerPlayerEntity> players, GameSpace gameSpace, JoinIntent intent) {
         var results = new Results();
 
-        var screenResult = gameSpace.getPlayers().screenJoins(players, intent);
-        if (screenResult.isError()) {
-            results.globalError = screenResult.error();
-            return results;
-        }
-
-        for (var player : players) {
-            var result = gameSpace.getPlayers().offer(player, intent);
-            if (result.isError()) {
+        var result = gameSpace.getPlayers().offer(players, intent);
+        if (result.isError()) {
+            for (var player : players) {
                 results.playerErrors.put(player, result.error());
```

Comment:
Suggestion: we can probably simplify this structure a bunch now, as we only get one error anyway :slightly_smiling_face:

Reply:
Ok, replacing `GamePlayerJoiner.Results` with `GameResult` seems to be a good idea.